### ISSUE TITLE
Chrome [HTML5] Video Tag does not allow to control a video

### ISSUE THREAD

**Issue by BoQsc** [DannyArends/DaNode#29]
Low importance issue.

[issue's title says it all, look up the video below.]

> Somehow I suspect that this has to do with the Web Server. (DaNode)  
> Haven't tested on Apache or other Web Servers yet.  
> But it (HTML video tag) seems to **work perfectly for everyone else that do not use DaNode**.  

https://user-images.githubusercontent.com/21064622/146768482-66840ea5-c33b-4669-84ab-a2eff3075a0f.mp4

Guide to reproduce
[Temporary live version. (1-2 days)](http://78.140.136.25/index2.html)

1. This is what I've used.
```
<video width="320" height="240" controls>
  <source src="1.mp4" type="video/mp4">
  <source src="1.ogg" type="video/ogg">
Your browser does not support the video tag.
</video>
```

2. Now place a video .mp4 file named 1.mp4 in the same folder alongside.
3. Place it in the DaNode www/*/ folder
4. Test things out and see that you can't control the video.

**Comment by DannyArends** [comment]
Probably has something to do with the player requesting the file in chunks.

In theory the browser asks for Transfer-Encoding field set to "chunked", the server currently doesn't do chunked (it could though).

DaNode should respond appropriately I guess.

Could you capture some of the HTML requests ?

**Comment by BoQsc** [comment]
> Could you capture some of the HTML requests ?

### HTML requests 
Below this image there are exact values.

![image](https://user-images.githubusercontent.com/21064622/146792642-ca3bb90b-2654-499f-9bc7-f0178c90dd27.png)



Values from the image above in the same order:
<table>
  <tr>
    <th>GET</th>
    <th>Response</th>
  </tr>
  <tr>
    <td> <pre><code>
Host: 78.140.136.25
DNT: 1
Accept-Encoding: identity;q=1, *;q=0
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
Range: bytes=0-
Accept: */*
Referer: http://78.140.136.25/index2.html
Accept-Language: en-US,en;q=0.9,lt-LT;q=0.8,lt;q=0.7
</pre> </code> </td>
    <td> <pre><code>
HTTP/1.1 200 Ok
X-Powered-By: Digital Mars D 2.98
Server: DaNode/0.0.3
Date: 20 Dec 2021 10:30:23 CET
Content-Length: 7328150
Last-Modified: 19 Dec 2021 7:10:22 CET
Content-Type: video/mp4
Connection: Keep-Alive

</pre> </code> </td>

  </tr>
  <tr>
    <td><pre><code>
Host: 78.140.136.25
DNT: 1
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.9,lt-LT;q=0.8,lt;q=0.7

</code></pre></td>
    <td><pre><code>
HTTP/1.1 200 Ok
X-Powered-By: Digital Mars D 2.98
Server: DaNode/0.0.3
Content-Encoding: deflate
Date: 20 Dec 2021 10:30:23 CET
Content-Length: 396
Last-Modified: 20 Dec 2021 7:25:13 CET
Content-Type: text/html
Connection: Keep-Alive

</code></pre></td>
  </tr>
</table>

Note: [Capture software](https://chrome.google.com/webstore/detail/http-repeater-https-live/jhojhhcdnopmbacgkggcgldbclfneefd/related?hl=en)

**Comment by DannyArends** [comment]
Probably due to the 

    Range: bytes=0-

Header set/requested, I cannot reproduce it under Firefox, .mp4 seems to play well and can be skipped

**Comment by BoQsc** [comment]
> Header set/requested, I cannot reproduce it under Firefox, .mp4 seems to play well and can be skipped

That's true, Firefox does do everything properly. Only Chrome seemingly affected

**Comment by DannyArends** [comment]
@BoQsc (Sorry to necro this issue) This should be implemented now, thanks for the suggestion.
